content/en/logs/log_configuration/online_archives.md

```diff
@@ -49,12 +51,17 @@ Configure Online Archives in the [Logs Index Configuration][3] page:
 
 ## Searching in Online Archives
 
-Select Online Archives from the dropdown in the Logs Explorer to begin searching in Online Archives instead of indexes. Find this dropdown next to the time picker.
+Select Online Archives from the dropdown in the Logs Explorer to begin searching in Online Archives instead of indexes. Find this dropdown next to the time picker. You can adjust the time picker with pre-set options, such as 2 or 3 months, or by selecting the calendar view to search further back in time.
```

Comment:
```suggestion
Select Online Archives from the dropdown in the Logs Explorer to begin searching in Online Archives instead of indexes. Find this dropdown next to the time picker. You can adjust the time picker by selecting pre-set options, up to 3 months, or by selecting the calendar view to search further back in time.
```